src/tiled/automapper.cpp

```diff
@@ -631,8 +642,26 @@ static bool layerMatchesConditions(const TileLayer &setLayer,
                         !setLayer.contains(x + offset.x(), y + offset.y()))
                     return false;
 
-                const Cell &setCell = setLayer.cellAt(x + offset.x(),
-                                                      y + offset.y());
+                int xd = x + offset.x();
+                int yd = y + offset.y();
```

Comment:
Now that we have these variables, maybe move them up so that you can also use them in the `contains` check.

Reply:
Moved up.